crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -137,6 +137,190 @@ pub struct StandardMaterial {
     #[doc(alias = "specular_intensity")]
     pub reflectance: f32,
 
+    /// The amount of light transmitted _diffusely_ through the material (i.e. “translucency”)
+    ///
+    /// Implemented as a second, flipped [Lambertian diffuse](https://en.wikipedia.org/wiki/Lambertian_reflectance) lobe,
+    /// which provides an inexpensive but plausible approximation of translucency for thin dieletric objects (e.g. paper,
+    /// leaves, some fabrics) or thicker volumetric materials with short scattering distances (e.g. porcelain, wax).
+    ///
+    /// For specular transmission usecases with refraction (e.g. glass) use the [`StandardMaterial::transmission`] and
+    /// [`StandardMaterial::ior`] properties instead.
+    ///
+    /// - When set to `0.0` (the default) no diffuse light is transmitted;
+    /// - When set to `1.0` all diffuse light is transmitted through the material;
+    /// - Values higher than `0.5` will cause more diffuse light to be transmitted than reflected, resulting in a “darker”
+    ///   appearance on the side facing the light than the opposite side. (e.g. plant leaves)
+    ///
+    /// **Important:** The material's [`StandardMaterial::base_color`] also modulates the transmitted light.
+    ///
+    /// **Note:** Typically used in conjunction with [`StandardMaterial::thickness`]. To avoid self-shadows on complicated
+    /// mesh shapes without having to fine tune the thickness, consider using the [`NotTransmittedShadowReceiver`](crate::light::NotTransmittedShadowReceiver) component.
+    #[doc(alias = "translucency")]
+    pub diffuse_transmission: f32,
+
+    /// A map that modulates diffuse transmission via its **A channel**. Multiplied by [`StandardMaterial::diffuse_transmission`]
+    /// to obtain the final result.
+    ///
+    /// **Important:** The [`StandardMaterial::diffuse_transmission`] property must be set to a value higher than 0.0,
+    /// or this texture won't have any effect.
+    ///
+    /// ## Texture Packing
```

Comment:
I don't understand what the texture packing section implies. Was this meant to be an internal comment, and not an end-user doc comment?

Reply:
This was meant as a hint to the developer on how to leverage the specific channel mapping that glTF mandates for these textures to achieve more efficient texture packing. Maybe it's too low-level/implementation detail to expose as a documentation comment? With the new Asset stuff it might not make as much sense to include this it's conceivable texture packing can be done automatically.

Reply:
Yeah maybe just remove it for now and make a note in the PR description.

Reply:
Removed it, added as a note in the PR